content/blog/2020-03-forcats-0-5-0/index.Rmarkdown

```diff
@@ -0,0 +1,87 @@
+---
+title: forcats 0.5.0
+author: Mara Averick
+date: '2020-03-02'
+slug: forcats-0-5-0
+description: >
+    Announcing the release of forcats 0.5.0 on CRAN. 
+categories:
+  - package
+tags:
+  - tidyverse
+  - forcats
+photo:
+  url: https://unsplash.com/photos/YCPkW_r_6uA
+  author: Jari Hytönen
+---
+
+```{r setup, include = FALSE}
+library(tidyverse)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're exceedingly happy to announce the release of [forcats](https://forcats.tidyverse.org/) 0.5.0 on CRAN.
+The goal of the forcats package is to provide a suite of tools that solve common problems with factors, including changing the order of levels or the values.
+
+This release includes improvements to several existing functions, as well as a division of [`fct_lump()`](https://forcats.tidyverse.org/reference/fct_lump.html) into four new functions: `fct_lump_min()`, `fct_lump_prop()`, `fct_lump_n()`, and `fct_lump_lowfreq()`. For a complete inventory of updates in this version, please see the [Change log](https://forcats.tidyverse.org/dev/news/index.html).
+
+You can install forcats with:
+
+```{r install, eval = FALSE}
+install.packages("forcats")
+```
+
+Attach the package by running:
+
+```{r forcats}
+library(forcats)
+```
+
+## New features
+
+### `fct_lump()` function family
+
+Though `fct_lump()` will continue to exist, it has been split into a family of four functions for lumping together levels that meet some criteria as "Other."  
+
+  * `fct_lump_min()` lumps levels that appear fewer than `min` times.  
+  * `fct_lump_prop()` lumps levels that appear fewer than `prop * n` times.  
+  * `fct_lump_n()` lumps all levels except for the `n` most frequent (or least frequent, if `n < 0`).  
+  * `fct_lump_lowfreq()` lumps together the least frequent levels, ensuring that `"Other"` is still the smallest level.  
+
+<br/><br/>
+
+```{r fct-lump-setup}
+x <- factor(rep(LETTERS[1:8], times = c(40, 10, 5, 27, 3, 1, 1, 1)))
+
+x %>% table()
+```
+
+
+```{r fct-lump-min}
+x %>% fct_lump_min(5) %>% table()
+```
+
+
+```{r fct-lump-prop}
+x %>% fct_lump_prop(0.10) %>% table()
+```
+
+
+```{r fct-lump-n}
+x %>% fct_lump_n(3) %>% table()
+```
+
+```{r}
+x %>% fct_lump_lowfreq() %>% table()
+```
+
+### New arguments, and helpers
+
+[`fct_collapse()`](https://forcats.tidyverse.org/reference/fct_collapse.html) now has an argument, `other_level`, which allows a user-specified `Other` level. Factors are now correctly collapsed when `other_level` is not `NULL`, and makes `Other` the last level.
+
+[`fct_reorder2()`](https://forcats.tidyverse.org/reference/fct_reorder.html) now has a helper function, `first2()`, which sorts `.y` by the first value of `.x`. 
+
+## Acknowledgements
+
+We're extremely grateful for the 48 people who contributed to this release:
```

Comment:
Would you mind specifically calling out TDD? e.g. https://github.com/tidyverse/forcats/pulls?q=is%3Apr+is%3Aclosed+label%3A%22tidy-dev-day+%3Anerd_face%3A%22 (but you'll need to filter by date too)